backend/models/class.model.ts

```diff
@@ -22,9 +22,13 @@ export interface Class extends Document {
 }
 
 export interface Student {
+  /** the unique identifier for the student */
   id: string;
+  /** the first name of the student */
   firstName: string;
+  /** the last name of the student */
   lastName: string;
+  /** an optional identifier provided by the teacher */
```

Comment:
is this unique as well?

Reply:
uhhh sort of? I guess it would probably be a unique input provided by the teacher (i.e no two students in the class would have the same identifier), but there is probably no guarantee of "uniqueness" across our system, since theoretically different teachers could have the same identifiers for different students (i.e if they number them #1, #2, etc.). so i hesitate to call it a `unique identifier`, but let me know if there is some way i can update the documentation to be more clear